halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentEnvironment.java

```diff
@@ -94,6 +94,7 @@ public static Size fromString(String name) {
   private String accountName;
   private Consul consul = new Consul();
   private Vault vault = new Vault();
+  private String location = "spinnaker";
```

Comment:
@lwander Are you recommending that we:
* leave it unset here;
* set the "spinnaker" default only in the k8s and openstack providers;
* set sane defaults in all the other providers?